README.md

```diff
@@ -145,9 +169,10 @@ docker compose -f docker-compose.yml -f docker-compose-maintenance.yml up flyway
 ## Publish Docker Images
 
 To publish the Docker images to a registry, first build the worker base images as described in
-[Local Setup](#local-setup). Then you can use the `jib` task to publish the images by setting the correct prefix for the
-registry. You can also configure the tag which defaults to `latest`. When publishing multiple images at once it is
-recommended to disable parallel builds.
+[Running with local images](#running-with-local-images).
+Then you can use the `jib` task to publish the images by setting the correct prefix for the registry.
+You can also configure the tag which defaults to `latest`.
+When publishing multiple images at once, it is recommended to disable parallel builds.
```

Comment:
Maybe write a line to explain how to disable parallel builds ?

Reply:
I have removed the sentence instead, see the commit message.

Reply:
Ah ok. Anyway I missed the lines underneath with `no-parallel`. :see_no_evil: